easybuild/easyconfigs/m/MiGEC/MiGEC-1.2.8-Java-1.8.0_92.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'MiGEC'
+version = '1.2.8'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://milaboratory.com/software/migec/'
+description = """ MIGEC is a software pipeline that facilitates processing and analysis of 
+ immune repertoire sequencing data generated using molecular barcoding technique"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/mikessh/migec/releases/download/%(version)s/']
+sources = ['%(namelower)s-%(version)s.zip']
+
+dependencies = [('Java', '1.8.0_92')]
```

Comment:
Any reason you're not using the latest `Java` (`1.8.0_162`)?

Reply:
I have used this version because it's the latest one in my cluster. I can bump up the version if you prefer